README.md

```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white) ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
 
-## Available Scripts
 
-In the project directory, you can run:
+# Math Magicians 🧮
+> React Web App for math lovers.
 
-### `npm start`
+## Tech ⌘⇧
+- **HTML & CSS best practices:** Correct use of tags, elements, properties and syntax.
+- **GitHub flow:**  Correct use of Branches for deployment and features development.
+- **Webpack:**  Source development for production bundling.
+- **Linters Check:** Local and Pull Request Github workflows to check for stylistic errors, bugs and syntax errors in code.
+- **React:** Build on `React.js` framework with jsx.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Clone Repository
+- Copy this code on your CLI to clone the repository `$ git clone https://github.com/raminka13/math-magicians.git`
+- `cd math-magicians`
+- run `npm install` to install all the dependencies.
+- run `npm start` to open the app in your localServer.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+## Author ✍🏼
+👤 **Raul A Ospina** 🧑🏻‍💻
+- GitHub: [@raminka13](https://github.com/raminka13)
+- Twitter: [@raminka13](https://twitter.com/raminka13)
+- LinkedIn: [Raul Ospina](http://linkedin.com/in/raul-ospina-83232614)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Project Milestones
+- **Milestone 0 - project setup (0.5h)** ✅
+- **Milestone 1 - basic React components (1h)** ✅
+- **Milestone 2 - Calculator logic with events (1h)** 
+- **Milestone 3 - Refactor with hooks (1h)** 
 
-### `npm run build`
+## Show your support 🦾
+- Give a ⭐️ if you like this project!
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add the **:handshake: Contributing** section from the **[template](https://github.com/microverseinc/readme-template)** to help users who find some bugs in your project to mentions them.